packages/components/document-card/src/document-card.css

```diff
@@ -0,0 +1,53 @@
+/* General........................................................ */
+
+:host {
+	overflow: hidden;
+	white-space: nowrap;
+}
+
+.document-card {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+}
+
+.card-container {
+	overflow: hidden;
+	white-space: nowrap;
+	padding-left: 5px;
+}
+
+.card-icon {
+	padding: 5px;
+}
+
+.card-title {
+	font-weight: var(--ts-fontweight-bold);
+	padding-bottom: 5px;
+	color: #192226;
+	font-size: var(--ts-fontsize);
+	text-overflow: ellipsis;
+	overflow: hidden;
+}
+
+.card-description,
+.card-mobile-description {
+	color: #192226;
```

Comment:
Please, use colours from the `packages/core/src/vars.css`, there are colours approved by our designers.

Reply:
ok